chemprop/sklearn_train.py

```diff
@@ -105,7 +105,11 @@ def single_task_sklearn(model: Union[RandomForestRegressor, RandomForestClassifi
             dataset_type=args.dataset_type,
             logger=logger
         )
-        scores.append(score[0])
+       
+        for metric in metrics:
+            if metric not in scores.keys():
```

Comment:
scores.keys() creates a list of dictionary keys, and it is usually preferred to just do `if metric not in scores` instead